src/components/AgreementSections/Info.js

```diff
@@ -62,6 +62,19 @@ const Info = ({ agreement }) => {
           </KeyValue>
         </Col>
       </Row>
+      {agreement?.licenseNote ?
```

Comment:
I think elsewhere we use the pattern
```
agreement?.licenseNote ?? <content />
```

Other than that, seems good

Reply:
It's 
`agreement?.licenseNote && <content />`